servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/TopicDescription.java

```diff
@@ -159,18 +155,18 @@ public Duration getDuplicationDetectionHistoryTimeWindow() {
      * Max value is 1 day and minimum is 20 seconds.
      */
     public void setDuplicationDetectionHistoryTimeWindow(Duration duplicationDetectionHistoryTimeWindow) {
-        if (duplicationDetectionHistoryTimeWindow != null &&
-                (duplicationDetectionHistoryTimeWindow.compareTo(ManagementClientConstants.MIN_DUPLICATE_HISTORY_DURATION) < 0 ||
-                        duplicationDetectionHistoryTimeWindow.compareTo(ManagementClientConstants.MAX_DUPLICATE_HISTORY_DURATION) > 0))
-        {
+        if (duplicationDetectionHistoryTimeWindow != null
+            && (duplicationDetectionHistoryTimeWindow.compareTo(ManagementClientConstants.MIN_DUPLICATE_HISTORY_DURATION) < 0
+                || duplicationDetectionHistoryTimeWindow.compareTo(ManagementClientConstants.MAX_DUPLICATE_HISTORY_DURATION) > 0)) {
             throw new IllegalArgumentException(
                     String.format("The value must be between %s and %s.",
                             ManagementClientConstants.MIN_DUPLICATE_HISTORY_DURATION,
                             ManagementClientConstants.MAX_DUPLICATE_HISTORY_DURATION));
         }
 
         this.duplicationDetectionHistoryTimeWindow = duplicationDetectionHistoryTimeWindow;
-        if (this.duplicationDetectionHistoryTimeWindow.compareTo(ManagementClientConstants.MAX_DURATION) > 0) {
+        if (this.duplicationDetectionHistoryTimeWindow != null
```

Comment:
[SpotBugs-P2] Possible null pointer dereference of duplicationDetectionHistoryTimeWindow in com.microsoft.azure.servicebus.management.TopicDescription.setDuplicationDetectionHistoryTimeWindow(Duration)